aea/configurations/schemas/definitions.json

```diff
@@ -113,6 +113,10 @@
     "execution_timeout": {
       "type": ["number", "null"],
       "minimum": 0
+    },
+    "skill_exception_policy": {
+      "type": ["string", "null"],
+      "minimum": 0
```

Comment:
could restrict to 3 available policies

Reply:
Shall we use enums?
https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values